src/renderer/components/kube-object/kube-object-menu.tsx

```diff
@@ -7,13 +7,13 @@ import { hideDetails } from "./kube-object-details";
 import { apiManager } from "../../api/api-manager";
 import { kubeObjectMenuRegistry } from "../../../extensions/registries/kube-object-menu-registry";
 
-export interface KubeObjectMenuProps<T extends KubeObject = any> extends MenuActionsProps {
-  object: T;
+export interface KubeObjectMenuProps<T> extends MenuActionsProps {
+  object: T | null | undefined;
```

Comment:
Why both `null` and `undefined`?  
Can you provide an example when it could be helpful? since `null == undefined` // true

Reply:
I guess I can remove one of them, I did this to be "most correct" since we sometimes return `null` and sometimes return `undefined`. When we move to `strict` mode this will become more relevant.

Reply:
Are we going to move forward `strict`? What's for if so?

Reply:
It help catches more problems, the rules that it turns on is:

- noImplicitAny
- noImplicitThis
- strictNullChecks
- strictPropertyInitialization
- strictBindCallApply
- strictFunctionTypes